changes/348.general.rst

```diff
@@ -0,0 +1 @@
+Updated ``build_driz_weight`` to raise an exception when requested ``weight_type`` is not supported.
```

Comment:
Would you remove this changelog entry? It's no longer accurate and I don't think this PR needs a changelog for just unit test changes.

Reply:
I removed it